apps/aecore/lib/aecore/channel/worker.ex

```diff
@@ -67,6 +67,11 @@ defmodule Aecore.Channel.Worker do
     closed(tx)
   end
 
+  def new_tx_mined(%SignedTx{data: %DataTx{type: ChannelSnapshotSoloTx}} = _tx) do
+    # TODO
```

Comment:
I think we shouldn't do anything in this case (replace the comment with `SnapshotTx requires no action`)